cypress/TestCases/FireboltCertification/Accessibility.feature

```diff
@@ -5,8 +5,8 @@ Feature: Accessibility
         Given the environment has been set up for 'Accessibility' tests
         And 3rd party 'certification' app is launched
 
-    @sdk @transport
-    Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario>
+    @sdk @transport @Sev0
+    Scenario Outline: Accessibility.closedCaptionsSettings - Validating API and Event Responses for <Scenario>
         Given we test the 'ACCESSIBILITY_CLOSEDCAPTIONS_SETTINGS' getters and setters '<Method>' to '<Value>'
```

Comment:
Just curious on the scenario names. I thought we have to Omit Validate/Validating in our test cases ?